modules/custom/az_person/config/install/views.view.az_person.yml

```diff
@@ -377,13 +375,139 @@ display:
           multi_type: separator
           separator: ', '
           field_api_classes: false
-          plugin_id: field
+      sorts:
+        weight:
+          id: weight
+          table: draggableviews_structure
+          field: weight
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: standard
+          order: ASC
+          expose:
+            label: ''
+            field_identifier: weight
+          exposed: false
+          draggable_views_reference: 'az_reorder:reorder_people'
+          draggable_views_null_order: after
+          draggable_views_pass_arguments: 0
+        field_az_lname_value:
+          id: field_az_lname_value
+          table: node__field_az_lname
+          field: field_az_lname_value
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: standard
+          order: ASC
+          expose:
+            label: ''
+            field_identifier: field_az_lname_value
+          exposed: false
+        field_az_fname_value:
+          id: field_az_fname_value
+          table: node__field_az_fname
+          field: field_az_fname_value
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: standard
+          order: ASC
+          expose:
+            label: ''
+            field_identifier: field_az_fname_value
+          exposed: false
       defaults:
-        fields: false
         relationships: false
+        fields: false
         sorts: false
         arguments: true
       relationships: {  }
+      display_description: ''
+      display_extenders: {  }
+      path: people
+    cache_metadata:
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url
+        - url.query_args
+        - 'user.node_grants:view'
+        - user.permissions
+      tags:
+        - 'config:field.storage.node.field_az_person_category'
+  grid_3col:
+    id: grid_3col
+    display_title: 'Grid - 3 Col'
+    display_plugin: block
```

Comment:
Why was this added as a block and not a Page like Grid Page and Row Page?

I think this display option should be consistent with the other existing options and be its own Page instead of a Block.

Reply:
If it's a Page, it requires a path and builds the entire page. This is not meant to replace the default page, but rather to provide a three-col option.

Reply:
Yes, you can make it a page just like Grid Page and Row Page without replacing the 'default' page.

Grid Page exists at: /people
Row Page exists at: /people-list

You can keep this new grid_3col display consistent with those two existing display options by using their precedent and making it a page as well.

Grid - 3 Col could exist at: /people-3col

Then, this new display should also be added to demo content's menu.

Reply:
Yes, thanks. We discussed in the PR review this a.m. and agreed that Block was suitable. Working now on getting the Demo version in the menu and the weird Master changes cleaned up.

Reply:
Could you please explain how block is suitable when the precedent set by the other displays is to use page?

Reply:
Block is more flexible and allows editors to set their URL via the page settings rather than setting it in the view. This is meant to be another option, not to replace existing functionality. So the block is the addition.

Reply:
There is no suggestion here to replace existing functionality. Adding a page is an _addition_, not a _replacement_. I'm not sure why you keep thinking a page would replace anything.

Yes, block can be more flexible, but is not the standard already set by the other pages. If block is the new format to use, the /people-list should be converted to a block as well.